docs/user/get_started.md

```diff
@@ -0,0 +1,233 @@
+from garage.torch.algos import TRPO as PyTorch_TRPO
+from garage.torch.policies import GaussianMLPPolicy as PyTorch_GMP
+from garage.torch.value_functions import GaussianMLPValueFunction
+
+hyper_parameters = {
+    'hidden_sizes': [32, 32],
+    'max_kl': 0.01,
+    'gae_lambda': 0.97,
+    'discount': 0.99,
+    'max_episode_length': 100,
+    'n_epochs': 999,
+    'batch_size': 1024,
+}
+
+@wrap_experiment
+def trpo_garage_pytorch(ctxt, env_id, seed):
+    deterministic.set_seed(seed)
+
+    runner = LocalRunner(ctxt)
+
+    env = GarageEnv(normalize(gym.make(env_id)))
+
+    policy = PyTorch_GMP(env.spec,
+                         hidden_sizes=hyper_parameters['hidden_sizes'],
+                         hidden_nonlinearity=torch.tanh,
+                         output_nonlinearity=None)
+
+    value_function = GaussianMLPValueFunction(env_spec=env.spec,
+                                              hidden_sizes=(32, 32),
+                                              hidden_nonlinearity=torch.tanh,
+                                              output_nonlinearity=None)
+
+    algo = PyTorch_TRPO(
+        env_spec=env.spec,
+        policy=policy,
+        value_function=value_function,
+        max_episode_length=hyper_parameters['max_episode_length'],
+        discount=hyper_parameters['discount'],
+        gae_lambda=hyper_parameters['gae_lambda'])
+
+    runner.setup(algo, env)
+    runner.train(n_epochs=hyper_parameters['n_epochs'],
+                 batch_size=hyper_parameters['batch_size'])
+```
+
+[This page](https://garage.readthedocs.io/en/latest/user/experiments.html) will give you more insight into running experiments.
+
+### Plotting results
+
+In garage, we use [TensorBoard](https://www.tensorflow.org/tensorboard) for plotting experiment results.
+
+[This guide](https://garage.readthedocs.io/en/latest/user/monitor_experiments_with_tensorboard.html) will provide details how to set up tensorboard when running experiments in garage.
+
+### Experiment outputs
+
+`Localrunner` is a state manager of experiments in garage, It is set up to create, save and restore the state, also known as `snapshot` object, upon/ during an experiment. The `snapshot` object includes the hyperparameter configuration, training progress, a pickled object of algorithm(s) and environment(s), tensorboard event file etc.
+
+Experiment results will, by default, output to the same directory as the garage package in the relative directory `data/local/experiment`. The output directory is generally organized as the following:
+
+```bash
+└── data
+    └── local
+        └── experiment
+            └── your_experiment_name
+                ├── progress.csv
+                ├── debug.log
+                ├── variant.json
+                ├── metadata.json
+                ├── launch_archive.tar.xz
+                └── events.out.tfevents.xxx
+```
+
+You can modify the default output directory by modifying `snapshot_dir` in `src/garage/experiment/snapshotter.py`.
```

Comment:
This isn't so helpful. It would me more helpful to mention the arguments to `wrap_experiment` (e.g. `log_dir`).